reservation/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls        import path,include
+from reservation.views  import ReservationListView
+
+urlpatterns = [
+    path('/list', ReservationListView.as_view())
```

Comment:
```suggestion
    path('/', ReservationListView.as_view())
```

url이 reservation으로 끝나는 것 만으로 예약 목록을 표출한다고 고려할 수 있으므로 `list`라는 단어는 꼭 필요하지 않습니다 :) url은  최대한 resource 이름만으로 간결하게 관리해주세요 👍

Reply:
넵!!